webpack/webpack.renderer.config.js

```diff
@@ -0,0 +1,263 @@
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const tsImportPluginFactory = require("ts-import-plugin");
+const webpack = require("webpack");
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const packageJson = require("../package.json");
+
+const { devServer } = require("./constants");
+
+module.exports = ({ production })  => {
+  const mode = production ? 'production' : 'development';
+
+  return {
+    context: path.resolve(__dirname, '..'),
+    devtool: 'eval-source-map',
+    externals:
```

Comment:
Ideally we would do some DRY between this file and `webpack.renderer.js` with a shared `webpack.common.js`

Reply:
Should be simple, but I made a separate ticket for it + will make a separate PR
https://github.com/aics-int/aics-file-upload-app/issues/86